src/jl_uv.c

```diff
@@ -1037,11 +1107,108 @@ JL_DLLEXPORT int jl_queue_work(work_cb_t work_func, void *work_args, void *work_
     baton->notify_func = notify_func;
     baton->notify_idx = notify_idx;
 
+    JL_UV_LOCK();
     uv_queue_work(jl_io_loop, &baton->req, jl_work_wrapper, jl_work_notifier);
+    JL_UV_UNLOCK();
 
     return 0;
 }
 
+JL_DLLEXPORT void jl_uv_stop(uv_loop_t* loop)
+{
+    JL_UV_LOCK();
+    uv_stop(loop);
+    // TODO: use memory/compiler fence here instead of the lock
+    JL_UV_UNLOCK();
+}
+
+JL_DLLEXPORT void jl_uv_update_time(uv_loop_t* loop)
+{
+    JL_UV_LOCK();
+    uv_update_time(loop);
+    JL_UV_UNLOCK();
+}
+
+JL_DLLEXPORT int jl_uv_timer_start(uv_timer_t* handle, uv_timer_cb cb,
+        uint64_t timeout, uint64_t repeat)
+{
+    JL_UV_LOCK();
+    int r = uv_timer_start(handle, cb, timeout, repeat);
+    JL_UV_UNLOCK();
+    return r;
+}
+
+JL_DLLEXPORT int jl_uv_timer_stop(uv_timer_t* handle)
+{
+    JL_UV_LOCK();
+    int r = uv_timer_stop(handle);
+    JL_UV_UNLOCK();
+    return r;
+}
+
+JL_DLLEXPORT int jl_uv_fs_scandir(uv_loop_t* loop, uv_fs_t* req, const char* path, int flags,
+	    uv_fs_cb cb)
```

Comment:
You've got a tab here and on most of the multi-line signatures below

Reply:
thanks, hopefully fixed everywhere